src/fetchers.js

```diff
@@ -382,30 +388,30 @@ function fetchers(params = {}) {
     contentOpts.GITHUB_TOKEN = githubToken;
   }
 
-  const staticResolver = resolveRef(staticOpts, log);
-  const contentResolver = equalRepository(staticOpts, contentOpts)
-    ? staticResolver
-    : resolveRef(contentOpts, log);
-
-  const staticPromise = updateOpts(staticOpts, staticResolver);
-  const contentPromise = updateOpts(contentOpts, contentResolver);
-
   const wskOpts = {
     // eslint-disable-next-line no-underscore-dangle
     __ow_headers: params.__ow_headers,
```

Comment:
I wouldn't overwrite wsk reserved names....

Reply:
I did not change that, was there before and that's how we pass header to content action already.

Reply:
never mind.... those are not sent using `action.invoke`... I think we are good.